.github/workflows/tuist.yml

```diff
@@ -78,13 +62,15 @@ jobs:
             'generate-3',
             'generate-4',
             'generate-5',
+            'generate-6',
             'init',
             'lint-project',
             'lint-code',
             'scaffold',
             'up',
             'build',
             'test',
+            'env',
```

Comment:
Should not `doc` added be here, too?

Reply:
I removed `doc` from now from the test-suite until I can think of a way to implement it without any inherent flakiness.